templates/unifi-poller.xml

```diff
@@ -3,25 +3,23 @@
   <Name>UniFi-Poller</Name>
   <Repository>golift/unifi-poller</Repository>
   <Registry>https://hub.docker.com/r/golift/unifi-poller/</Registry>
+  <Shell>sh</Shell>
   <Network>bridge</Network>
   <Privileged>false</Privileged>
-  <Support>https://hub.docker.com/r/golift/unifi-poller/</Support>
-  <Project>https://github.com/davidnewhall/unifi-poller</Project>
-  <Overview>Collect ALL UniFi Controller, Device and Client Data - Export to InfluxDB or Prometheus. Visualize with Grafana using included dashboards!&#xD;
-&#xD;
-This template works with poller versions 1.6.2+&#xD;
-&#xD;
-READ THE INSTRUCTIONS! &#xD;
-https://github.com/davidnewhall/unifi-poller/wiki/Installation</Overview>
-  <Category>Tools:Utilities Network:Management </Category>
+  <Changes>See https://github.com/unifi-poller/unifi-poller/wiki/CHANGES</Changes>
+  <Support>https://community.ui.com/questions/UniFi-Poller-Store-UniFi-Controller-Metrics-in-Prometheus-or-InfluxDB/58a0ea34-d2b3-41cd-93bb-d95d3896d1a1</Support>
+  <Project>https://github.com/unifi-poller/unifi-poller</Project>
+  <Category>Tools:Utilities Network:Management</Category>
   <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/unifi-poller.xml</TemplateURL>
-  <Icon>https://raw.githubusercontent.com/wiki/davidnewhall/unifi-poller/images/unifi-poller-logo.png</Icon>
-  <Description>Collect ALL UniFi Controller, Device and Client Data - Export to InfluxDB or Prometheus. Visualize with Grafana using included dashboards!&#xD;
-&#xD;
-This template works with poller versions 1.6.2+&#xD;
-&#xD;
-READ THE INSTRUCTIONS! &#xD;
-https://github.com/davidnewhall/unifi-poller/wiki/Installation</Description>
+  <Icon>https://raw.githubusercontent.com/wiki/unifi-poller/unifi-poller/images/unifi-poller-logo.png</Icon>
+  <Overview>
+    Collect ALL UniFi Controller, Device and Client Data - Export to InfluxDB or Prometheus.
+    Visualize with Grafana using included dashboards!
+    IMPORTAT! ACTION REQUIRED! As of UniFi Poller version 2 all of the environment variables and config file format changed.
```

Comment:
Can be highlighted
ref: https://github.com/selfhosters/unRAID-CA-templates/blob/master/templates/monitorr.xml#L10-L11

Reply:
That's a great idea, let me play with that!